src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettings.cs

```diff
@@ -1180,6 +1180,43 @@ public static bool RotaryEventEnabled
             }
         }
 
+        /// <summary>
+        /// This static member variable indicates whether OOBE(Out Of Box Experience) is enabled on the device.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/systemsettings.admin</privilege>
+        /// <privlevel>platform</privlevel>
+        /// <feature>http://tizen.org/feature/systemsetting</feature>
+        /// <feature>http://tizen.org/feature/systemsetting.oobe</feature>
+        /// <exception cref="ArgumentException">Invalid Argument</exception>
+        /// <exception cref="NotSupportedException">Not Supported feature</exception>
+        /// <exception cref="InvalidOperationException">Invalid operation</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+        /// <remarks>
+        /// http://tizen.org/privilege/systemsettings.admin is needed only for setting value. When getting the value, it isn't needed.
```

Comment:
The description is correct, but that would sound better IMO:
```suggestion
        /// http://tizen.org/privilege/systemsettings.admin is needed only for setting value. It is not required to get the value.
```